solidity/test/BondedECDSAKeepTest.js

```diff
@@ -986,46 +986,60 @@ contract("BondedECDSAKeep", (accounts) => {
         )
       }
     })
-  })
-
-  describe("slashSignerStakes", async () => {
-    it("is not public", async () => {
-      try {
-        await keep.slashSignerStakes()
-      } catch (err) {
-        assert.equal(err, "TypeError: keep.slashSignerStakes is not a function")
-      }
-    })
 
-    it("reverts if called by closed keep", async () => {
+    it("reverts if called for closed keep", async () => {
       await keep.publicMarkAsClosed()
 
-      await expectRevert(keep.publicSlashSignerStakes(), "Keep is not active")
+      await expectRevert(
+        keep.submitSignatureFraud(
+          signature1.V,
+          signature1.R,
+          signature1.S,
+          hash256Digest1,
+          preimage1
+        ),
+        "Keep is not active"
+      )
     })
 
-    it("reverts if called by terminated keep", async () => {
+    it("reverts if called for terminated keep", async () => {
       await keep.publicMarkAsTerminated()
 
-      await expectRevert(keep.publicSlashSignerStakes(), "Keep is not active")
+      await expectRevert(
+        keep.submitSignatureFraud(
+          signature1.V,
+          signature1.R,
+          signature1.S,
+          hash256Digest1,
+          preimage1
+        ),
+        "Keep is not active"
+      )
     })
 
-    it("slashes keep members stakes", async () => {
```

Comment:
Are we removing this test on purpose?

Reply:
Yes, this is not covered [here](https://github.com/keep-network/keep-ecdsa/blob/77b95f9e47b908dc30503e315aa04e80b2f0c8ce/solidity/test/BondedECDSAKeepTest.js#L932-L959).